static/js/entry/uswitch.js

```diff
@@ -0,0 +1,26 @@
+/* global SETTINGS: false */
+require('react-hot-loader/patch');
+const $script = require('scriptjs');
```

Comment:
I'm not sure we should use `$script` here. The below async imports depend on one another so that they're loaded in sequence instead of in parallel. If we include the script tags in the HTML right before the `render_bundle` for this entry we should get the same behavior and the script tags will load in parallel too